src/Storage/Helpers/ProcessHelper.cs

```diff
@@ -29,19 +29,19 @@ public static List<ProcessHistoryItem> MapInstanceEventsToProcessHistory(List<In
                         history.Add(
                             new ProcessHistoryItem
                             {
-                                Occured = instanceEvent.Created,
+                                Occured = instanceEvent?.ProcessInfo?.Started ?? instanceEvent.Created,
                                 EventType = instanceEvent.EventType,
-                                ElementId = instanceEvent.ProcessInfo.StartEvent,
+                                ElementId = instanceEvent?.ProcessInfo?.StartEvent,
```

Comment:
It seems as though not all instances have a ProcessInfo object, which would result in a null exception here, that is potentially being swallowed somewhere in the call stack.
Is it relevant to investigate this further?

Reply:
Don't think this is relevant to investigate further no.The instance events without processinfo are those not related to process steps but rather to enduser actions such as read and write.  No need for the null check either. if it is of the type `process_StartEvent` we can guarantee it isn't null